dgraphtest/cluster.go

```diff
@@ -21,29 +21,46 @@ import (
 	"fmt"
 	"io"
 	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
 	"time"
 
 	"github.com/docker/docker/api/types"
 	"github.com/docker/docker/api/types/container"
 	"github.com/docker/docker/api/types/network"
 	docker "github.com/docker/docker/client"
+	"github.com/go-git/go-git/v5"
+	"github.com/go-git/go-git/v5/plumbing"
 	"github.com/pkg/errors"
 )
 
 var (
-	requestTimeout = 30 * time.Second
-	stopTimeout    = time.Minute
+	requestTimeout    = 30 * time.Second
+	stopTimeout       = time.Minute
+	homePath          = os.Getenv("HOME") + "/data/dgraph"
```

Comment:
I think what we should do here is that, we should have a relative folder where we keep all the binaries. The name of the binaries could be something like `dgraph_{version}`. Then, every time we need to mount the binary inside the dir, we copy the binary into a temp folder (we can get it using os.TempDir) and then mount it inside the container. Doing this will ensure that we can run multiple clusters in parallel.